### DISCUSSION TITLE
Accessing Tailwind color palette value in tailwind.config.js

### DISCUSSION THREAD

**Discussion by dance2die** [tailwindlabs/tailwindcss#3785] [Help] [answered]
Is there a way to access TW color palette value like `color.green.600` in the configuration?

As I was helping out in https://github.com/tailwindlabs/tailwindcss/discussions/3767, I was looking up the hex value in the documentation, https://tailwindcss.com/docs/customizing-colors#color-palette-reference.

```js
const colors = require('tailwindcss/colors')

module.exports = {
  theme: {
    extend: {
      colors: {
        'light-blue': colors.lightBlue,
        cyan: colors.cyan,
        // color #16A34A is color.green.600 in
        // https://tailwindcss.com/docs/customizing-colors#color-palette-reference
        success: '#16A34A',
        warning: '#FBBF24',
        error: '#DC2626',
      },
    },
  },
}
```

Is there a way to access those hex values by name? I was using [theme()](https://tailwindcss.com/docs/functions-and-directives#theme) function but TW play was crashing.

**Comment by tehpsalmist** [comment, marked as answer]
Hey Sung! ;)

Did `colors.green['600']` not work?

**Reply by dance2die** [reply]
Thank you @tehpsalmist~ Worked great!

**Reply by tehpsalmist** [reply]
Have fun out there my friend!

**Reply by dance2die** [reply]
You too mate~ Thank you for introducing me to TW, my dear friend :)

**Comment by simonswiss** [comment]
Hey!

Here's how you can access the `colors.green['600']` value, as mentioned by @tehpsalmist just above 👍

https://play.tailwindcss.com/MYu3V76d7f?file=config

![image](https://user-images.githubusercontent.com/485747/111726492-312a8a00-88bd-11eb-9607-b2b3f08212fa.png)

**Reply by dance2die** [reply]
ty for the review and the demo~

**Reply by simonswiss** [reply]
No worries! 👍
